README.md

```diff
@@ -31,6 +31,8 @@ Xpring-JS is available as an NPM package. Simply install with:
 $ npm i xpring-js
 ```
 
+Xpring-JS can be used on the web with `webpack`, as it uses some Node modules. The sample `webpack.config.js` can be used as a starting point for web applications.
```

Comment:
Do we want to start recommending webpack as the preferred installation option?

If so we might want to add some more guidance for unfamiliar users. I'm sure it's user error on my part, but when I ran `npm run webpack`, I got these errors:

```
➜ npm run webpack

> xpring-js@5.0.1 webpack /Users/rgy/Git/xpringjs-repo
> npm run clean && ./scripts/regenerate_protos.sh && ./scripts/regenerate_pay_id_api.sh && npm run lint && tsc --noEmit && webpack && copyfiles -u 3 './src/XRP/Generated/**/*' ./build/XRP/Generated && copyfiles -u 3 './src/ILP/Generated/**/*' ./build/ILP/Generated


> xpring-js@5.0.1 clean /Users/rgy/Git/xpringjs-repo
> rm -rf ./src/XRP/Generated ./src/ILP/Generated ./dist ./build ./src/PayID/Generated

Generating Protocol Buffers from Rippled
./rippled/src/ripple/proto/: warning: directory does not exist.
Could not make proto path relative: ./rippled/src/ripple/proto//org/xrpl/rpc/v1/*.proto: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! xpring-js@5.0.1 webpack: `npm run clean && ./scripts/regenerate_protos.sh && ./scripts/regenerate_pay_id_api.sh && npm run lint && tsc --noEmit && webpack && copyfiles -u 3 './src/XRP/Generated/**/*' ./build/XRP/Generated && copyfiles -u 3 './src/ILP/Generated/**/*' ./build/ILP/Generated`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the xpring-js@5.0.1 webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/rgy/.npm/_logs/2020-07-28T22_17_51_402Z-debug.log
```

For now, I think the change would only affect the installation step of the tutorial ([here](https://xpring.io/docs/xpring-sdk/getting-started/#install-the-xpring-sdk-library)). Not sure if there's other stuff we'd want to change because of webpack.

Reply:
there's an additional step to generate protocol buffers, i believe! you have to checkout the submodules to get the build to go. alternatively, if you had a project downstream that was using a `webpack.config.js` that looked like this one, it should work

i don't think we should start recommending webpack as the preferred installation option - our code is based on the `node` implementation of `grpc`, and `webpack` is polyfilling `node` modules so that the code works in a browser environment. i propose instead we offer it as an option, and if we see more demand for a browser-solution, then we invest more time here - when that happens, i have a few ideas for how we could make this easier to consume downstream for users of xpring-js!